ros_webrtc/Constraint[] mandatory
ros_webrtc/Constraint[] optional
